### PR TITLE
Add non-technical instructions for Snowtype-gtm

### PR DESCRIPTION
**Pull request by igneel64**

### CODE REVIEW

File: docs/collecting-data/code-generation/working-with-gtm/index.md [hunk 1]
```diff
@@ -24,6 +28,36 @@ To generate code for usage in Google Tag Manager, you should use the option `Goo
...
+npm init -y
+# This will install the latest version of Snowtype.
+npm install @snowplow/snowtype@latest
+# This will start the Snowtype init flow, 
+# in which you should select 'Google Tag Manager' when 
+# prompted to select a tracker.
+npx @snowplow/snowtype init
+```
+
+After you have completed the `init` flow and have added your desired configuration, you can go ahead and generate the code you need to use.
```
Comment (gibbok-snowplow) [inline docs/collecting-data/code-generation/working-with-gtm/index.md:54]:
```suggestion
After you have completed the `init` flow and have added your desired configuration, you can go ahead and generate the code you need to use:
```

File: docs/collecting-data/code-generation/working-with-gtm/index.md [hunk 2]
```diff
@@ -24,6 +28,36 @@ To generate code for usage in Google Tag Manager, you should use the option `Goo
     "language": "javascript-gtm"
 }
 ```
+</details>
+
+What we are going to showcase here is how you can setup a separate project that uses Snowtype to generate code for you Google Tag Manager tracking needs. _You can also use version control to better understand any changes you do across time._
```
Comment (gibbok-snowplow) [inline docs/collecting-data/code-generation/working-with-gtm/index.md:33]: - "setup" should be "set up" in "how you can setup a separate project".
- "you Google Tag Manager tracking needs" should be "your Google Tag Manager tracking needs".
- "any changes you do across time" should be "any changes you make over time".
- "installed in your computer" should be "installed on your computer".
- Missing comma after "After that".
- "Change directory in the newly created folder" should be "Change directory to the newly created folder".
- "in which you should select 'Google Tag Manager' when prompted to select a tracker" could be made clearer with a slight rephrasing.

**Review by gibbok-snowplow — COMMENTED** [review]
Good job, I just added few minor thingy, please let me know if they make sense. Thanks

File: docs/collecting-data/code-generation/working-with-gtm/index.md (hunk 2 above)
Comment (igneel64) [reply docs/collecting-data/code-generation/working-with-gtm/index.md:33]: Done all of them except the last one since I cannot think of something better 😂

**Review by gibbok-snowplow — APPROVED** [review]
LGTM well done!

**Pull request merged**